### ISSUE TITLE
Clarification on age0 simData

### ISSUE THREAD

**Issue by vbartolino** [NOAA-EDAB/ms-keyrun#59]
Minimum age in the simulated datasets is age1 (which is the true age as far as I've understood), but I assume fish recruit in the OM at age0, correct?
Does it mean that no age0 is found in commercial or scientific survey?

**Comment by sgaichas** [comment]
This is a good question and important to clarify. 

I think Atlantis counts age 0 a bit differently than our usual convention; they will be in the first true age bin.

From the Atlantis manual (p. 184 in the Feb 2023 revision):
>Recruited individuals are attributed to the first year of the first age group. The day before the recruits “arrive”, all other individuals age one year up, and can be allocated into a different age group depending on the number of years in each age group. Ageing is done in one time step, regardless of whether recruits arrive all at once or during a certain period of time.

So Atlantis is calculating true age for each species, not a Jan 1 birthday age. 

Maybe I should provide the time (month) of recruit arrival in the model as well as the spawning month so that each modeler can apply an appropriate convention for ages? 

Month of recruitment along with the month that the survey or fishery takes place may help everyone decide if fish in Atlantis age 1 would have been age 0 prior to a Jan 1 birthdate convention.

**Comment by vbartolino** [comment]
This is very relevant for many reasons and I've the feeling it will have to be accounted for most of the estimation models (ie if you want to follow the same cohort between the two surveys...) and explains also the patterns mentioned in len@age and wgt@age #64. My preference (but it may be different for other estimation models) would be to align to the 1 Jan birthday convention and keep a calendar year in the model. This is the age relabelling that I'm considering for my estimation model:

COD: spawn month 1 recruit month 6 age1 --> age0 in the fall survey
HAD: spawn month 3 recruit month 6 age1 --> age0 in the fall survey
CAP:  spawn month 2 recruit month 14 age1 --> stay age1 in both spring and fall survey

I may be wrong but I've the impression that the age-based catch data need to be provided at a temporal resolution finer than the year to adjust for the same issue.

**Comment by vbartolino** [comment]
> Recruited individuals are attributed to the first year of the first age group. The day before the recruits “arrive”, all other individuals age one year up, and can be allocated into a different age group depending on the number of years in each age group. Ageing is done in one time step, regardless of whether recruits arrive all at once or during a certain period of time.

> So Atlantis is calculating true age for each species, not a Jan 1 birthday age.

What does it mean for the catch data? For instance, if in month 6 year 55 cod recruits as age1 will those fish move to age2 only in month 6 year 56? and to which period the fishery age comp in year 55 and year 56 will refer to?

**Comment by sgaichas** [comment]
> For instance, if in month 6 year 55 cod recruits as age1 will those fish move to age2 only in month 6 year 56?

Yes, I think so? I will confirm with the Atlantis modelers. But you are correct, I think for models tracking age in calendar years it would be most comparable with current practice to adjust these true Atlantis ages to calendar year ages using the rules you suggest above. I will definitely provide the catch age data at subannual timesteps so that these can be adjusted as well.

**Comment by vbartolino** [comment]
>Catch at age with subannual timesteps (months 3, 5, 8, 10, 12) is now provided on the sarah_wgsamsim branch [...]

Just a clarification, are these catch age compositions in the months 3, 5, 8, 10, 12 or in the timesteps 1-3, 4-5, 6-8, 9-10, 11-12?

**Comment by sgaichas** [comment]
I believe those are cumulative catch for the year at each output timestep, so month 3 would be catch at age in months 1-3, month 5 would be catch at age in 4-5, etc. They are actually 73 day outputs if you want exact timing:
# 1 is day 73, or 14 March; translates to month 3
# 2 is day 146, or 26 May, translates to month 5
# 3 is day 219, or 7 August, translates to month 8
# 4 is day 292, or 19 October, translates to month 10
# 5 is day 365, or 31 December, translates to month 12
